artifacthub/library/general/horizontalpodautoscaler/1.0.0/template.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: k8shorizontalpodautoscaler
+  annotations:
+    metadata.gatekeeper.sh/title: "Horizontal Pod Autoscaler"
+    metadata.gatekeeper.sh/version: 1.0.0
```

Comment:
can you add the requiresSyncData annotation? see examples in https://github.com/open-policy-agent/gatekeeper-library/pull/251

Reply:
absolutely, done 👍